src/esp/bindings/SensorBindings.cpp

```diff
@@ -107,7 +107,8 @@ void initSensorBindings(py::module& m) {
                                                                 "VisualSensor")
       .def(
           "get_render_camera", &VisualSensor::getRenderCamera,
```

Comment:
From my understanding, render_camera is an instance of RenderCamera, and we want to be able to get the object and call methods on it, rather than just get a value. You can see in GfxBindings the bindings for methods of RenderCamera

Reply:
Yeah, you can call methods  on a property. A property doesn't have to be a single value.